unepwcmc/action-agenda
# Feat/add UI for confiming cbd imports

We are pulling commitments from CBD. CFN does not want pledges to eat less meat under the supervision of mother to appear on the page. this adds an admin page where imported commitments can be quickly viewed and confirmed/rejected. 

Testing:
- run ``rails db:migrate:rest``` and ```rake import:commitments```
- sign in with test@test.com, which i snow an admin user
- should see an admin link in the nav drop down
- should see a list of the imported commitments
- accepting or rejecting a commitment should remove it, and it should then be viewable in the appropriate tab
- sign out and sign is as test2@test.com
- admin link should not be in navbar dropdown
- entering /admin/imported_commitments should send you to the home page
